packages/neon/neon/lib/src/pages/login_check_account.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:neon/l10n/localizations.dart';
+import 'package:neon/src/bloc/result.dart';
+import 'package:neon/src/bloc/result_builder.dart';
+import 'package:neon/src/blocs/accounts.dart';
+import 'package:neon/src/blocs/login_check_account.dart';
+import 'package:neon/src/models/account.dart';
+import 'package:neon/src/router.dart';
+import 'package:neon/src/widgets/account_tile.dart';
+import 'package:neon/src/widgets/exception.dart';
+import 'package:neon/src/widgets/linear_progress_indicator.dart';
+import 'package:neon/src/widgets/validation_tile.dart';
+import 'package:provider/provider.dart';
+
+class LoginCheckAccountPage extends StatefulWidget {
+  const LoginCheckAccountPage({
+    required this.serverURL,
+    required this.loginName,
+    required this.password,
+    super.key,
+  });
+
+  final String serverURL;
+  final String loginName;
+  final String password;
+
+  @override
+  State<LoginCheckAccountPage> createState() => _LoginCheckAccountPageState();
+}
+
+class _LoginCheckAccountPageState extends State<LoginCheckAccountPage> {
+  late final LoginCheckAccountBloc bloc;
+
+  @override
+  void initState() {
+    super.initState();
+
+    bloc = LoginCheckAccountBloc(
+      widget.serverURL,
+      widget.loginName,
+      widget.password,
+    );
+  }
+
+  @override
+  void dispose() {
+    bloc.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(final BuildContext context) => Scaffold(
+        appBar: AppBar(
+          leading: const CloseButton(),
```

Comment:
I might have wrongly communicated this. Sorry :)

I thought we only need the close button for the `LoginPage` (the initial page). As using it there will cancel the login. While using it here just goes back one step in the login flow (thus a backbutton might be prefered).

Reply:
No these pages also completely cancel the login, so this is correct

Reply:
You were right, my bad